queries.yaml

```diff
@@ -259,7 +259,7 @@ layers:
       - vectordatasource.transform.remove_feature_id
       - vectordatasource.transform.truncate_min_zoom_to_1dp
     sort: vectordatasource.sort.buildings
-    area-inclusion-threshold: 1
+    area-inclusion-threshold: 3
```

Comment:
Not sure what this is doing, let's prove it's having an effect +/- before changing